modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/query/RegisteredServiceQuery.java

```diff
@@ -16,14 +16,17 @@
  */
 @Data
 @Valid
-public class RegisterServiceQuery {
+public class RegisteredServiceQuery {
 
     @Schema(description = "Name of the cloud service provider.")
-    private String cspName;
+    private String csp;
```

Comment:
can this be the `Csp` Enum instead of string?

Reply:
Ok